NEWS.md

```diff
@@ -1,4 +1,43 @@
-## 3.1.0 - Unreleased
+## 4.1.0 - Unreleased
+
+## 4.0.0 - Released
```

Comment:
Maybe it makes sense to add short description about what was done before this release? Smth like implemented APIs: ...